.github/workflows/php-code-coverage.yml

```diff
@@ -0,0 +1,76 @@
+name: PHP code coverage
+
+on:
+  pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    name: PHP=8.1, WP=6.6, WC=9.1.2
+    env:
+      PHP_VERSION: '8.1'
+      WP_VERSION: '6.6'
+      WC_VERSION: '9.1.2'
+    steps:
+      - name: Get Changed Files
+        id: get-changed-files
+        uses: hanseltimeindustries/get-changed-files@v1
+        with:
+          format: 'space-delimited'
+          filter: '**/includes/**/*'
+      
+      - name: Checking out
+        if: ${{ steps.get-changed-files.outputs.all }} # Only run if there are changed files
```

Comment:
I tried to split this into different jobs, so we could have a single `if`, but that didn't work well and I decided to repeat the condition for now.